integration-tests/test-cases/scenario-tests/src/test/java/io/cellery/integration/scenario/tests/cli/ImageBasedTestCase.java

```diff
@@ -119,4 +119,10 @@ private void extractImage(String outputPath) throws Exception {
         String expectedOut = "Successfully extracted cell image resources: " + cellImageName;
         readOutputResult(process, expectedOut, errorString);
     }
+
+    @Test(dependsOnMethods = "extractResourceImageWithProvidedOutput")
+    public void deleteImages() throws Exception {
+        delete(Constants.TEST_CELL_ORG_NAME + "/" + EMPLOYEE_IMAGE + ":" + Constants.SAMPLE_CELLS_VERSION);
```

Comment:
Shall we move the repeating strings like / and : to constants?

Reply:
Updated.